packages/noir-ethereum-api/src/main.ts

```diff
@@ -36,6 +36,7 @@ export async function recordStorageProof(
   return isCorrect;
 }
 
+/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
```

Comment:
Why we need this? 🤔 It seems `proof` is used.

Reply:
We switched from `noir-js` proof verification to smart contract in `Solidity` verification. At this moment this method is not used. Added task to asana to discuss what kind proof verifications we want in e2e tests https://app.asana.com/0/1206416222603091/1206543007230561